test/unitTests/build/languageServerManager.test.ts

```diff
@@ -0,0 +1,100 @@
+import { SinonSandbox, createSandbox, SinonStub } from 'sinon';
+import { EnvironmentController } from '../../../src/common/environmentController';
+import { LanguageServerManager } from '../../../src/build/languageServerManager';
+import vscode from 'vscode';
+import { UserType } from '../../../src/shared';
+import { ExtensionActivated, StartLanguageServerCompleted, UserSignInCompleted } from '../../../src/common/loggingEvents';
+import assert from 'assert';
+
+describe('LanguageServerManager', () => {
+    let sinon: SinonSandbox;
+    let manager: LanguageServerManager;
+    let stubVSCodeExecuteCommand: SinonStub;
+    const fakeCorrelationId = 'fakeCorrelationId';
+
+    before(() => {
+        sinon = createSandbox();
+        stubVSCodeExecuteCommand = sinon.stub(vscode.commands, 'executeCommand');
+    });
+    afterEach(() => {
+        stubVSCodeExecuteCommand.reset();
+    });
+    after(() => {
+        stubVSCodeExecuteCommand.restore();
+    });
+
+    it('Real-time validation disabled', () => {
+        manager = new LanguageServerManager(<EnvironmentController>{
+            env: 'PROD',
+            docsRepoType: 'GitHub',
+            debugMode: false,
+            userType: UserType.MicrosoftEmployee,
+            enableAutomaticRealTimeValidation: false
+        });
+        manager.eventHandler(new ExtensionActivated());
+        assert(stubVSCodeExecuteCommand.withArgs('docs.enableRealTimeValidation').notCalled);
```

Comment:
consider move the `stubVSCodeExecuteCommand` to the initialization since it is the same for all the test